web-bundle/src/main/java/com/graphhopper/resources/MapboxResponseConverter.java

```diff
@@ -0,0 +1,324 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.resources;
+
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.graphhopper.GHResponse;
+import com.graphhopper.PathWrapper;
+import com.graphhopper.http.WebHelper;
+import com.graphhopper.util.*;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.UUID;
+
+public class MapboxResponseConverter {
+
+    // TODO: this is already in GraphHopper, but getting it from there is not possible easily?
+    private static final TranslationMap trMap = new TranslationMap().doImport();
```

Comment:
Do you mean hopper.getTranslationMap?

Reply:
Yes, but in the web-bundle we only have a reference to the GraphHopperAPI interface. We could probably check if the object is instance of GraphHopper and then get the TranslationMap? I thought that this would be too much overhead, but we could try?

Reply:
As we bind

> bind(translationMap).to(TranslationMap.class);

We could maybe inject it in the resource?

Reply:
Ah thanks, that's a good idea. I tried to inject it and it seems to work. Could you check if [this](https://github.com/graphhopper/graphhopper/pull/1422/files#diff-973e4c6fa0434d64f5e3b627775d7a19R274) is what you meant?